cli/clicommands/cmd.go

```diff
@@ -13,19 +13,19 @@ var Commands = []*cli.Command{
 	lcli.WithCategory("basic", lcli.MultisigCmd),
 	lcli.WithCategory("basic", lcli.FilplusCmd),
 	lcli.WithCategory("basic", lcli.PaychCmd),
-	lcli.WithCategory("developer", lcli.AuthCmd),
+	lcli.WithCategory("developer", lcli.AuthCmd()),
```

Comment:
why do these need to become functions?

Reply:
Previously, `lotus` and `lotus-miner` shared global command variables, causing conflicts when running app.Run("lotus-miner", "-h"). This led to jumbled help output like `lotus-miner lotus auth`.

Reply:
See below, I went for a bit of a trek to find out why this is the case and came up with an easier approach than rewriting all of these.